pkgs/development/libraries/agda/cubical/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, mkDerivation, fetchFromGitHub, ghc, glibcLocales }:
+
+mkDerivation rec {
+
+  # Version 0.2 is meant to be used with the Agda 2.6.1 compiler.
+  pname = "cubical";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "agda";
+    rev = "v${version}";
+    sha256 = "07qlp2f189jvzbn3aqvpqk2zxpkmkxhhkjsn62iq436kxqj3z6c2";
+  };
+
+  LC_ALL = "en_US.UTF-8";
```

Comment:
Does this not affect nearly every other Agda package? It feels like it should be the default.

Reply:
Maybe though we haven't run into this problem before. Also this only seems to arise on the haskell building of Everything files and I don't think it is to do with `agda` I'm not sure it is correct to add it to the builder

Reply:
Aah, okay, that explains a bit what's going on. I see stdlib's equivalent Haskell file makes sure everything is done in UTF-8, which is probably the right thing to do upstream.